core/src/script_builder.rs

```diff
@@ -105,7 +105,10 @@ impl ScriptBuilder {
         actor_taproot_address: &Address<NetworkUnchecked>,
         block_count: u32,
     ) -> ScriptBuf {
-        let actor_script_pubkey = actor_taproot_address.payload().script_pubkey();
+        let actor_script_pubkey = actor_taproot_address
+            .clone()
+            .assume_checked()
```

Comment:
it should not be a problem since actor address that has been used in this function is generally coming from a checked address, but we may change this to Address<NetworkChecked> in the future